src/transformers/models/auto/modeling_auto.py

```diff
@@ -579,6 +582,7 @@
         # Model for Image Classification mapping
         (ViTConfig, ViTForImageClassification),
         (DeiTConfig, (DeiTForImageClassification, DeiTForImageClassificationWithTeacher)),
+        (BEiTConfig, BEiTForImageClassification),
```

Comment:
I don't think so, since the model just predicts visual tokens (i.e. integers) from masked patches. This is not as exciting as letting BERT predict masked words.